checkpoint/postgres/postgres.go

```diff
@@ -0,0 +1,157 @@
+package postgres
+
+import (
+	"database/sql"
+	"fmt"
+	"sync"
+	"time"
+	// this is the postgres package so it makes sense to be here
+	_ "github.com/lib/pq"
+)
+
+// consumer represents the data that will be stored about the checkpoint
+type consumer struct {
+	Checkpoint JSONMap `db:"checkpoint"`
```

Comment:
wanted to get your take on using `jsonb` for 3 values; verses having a table w/ 3 columns?

Reply:
Hello I just thought it would be more simple to the client define when using the lib because its only one column and its not important the schema and it could be a little bit more flexible. Altough if you think it could be improve readability or it will be cleaner I will change it for a traditional table.

Reply:
yeah i'm on the fence, it feels like the cols will be pretty static (haven't needed to change in this repo ever), and it might clean up the SQL a little we wouldn't need to cast as JSON etc.

```
cols
----------------------------------
namespace
shard_id
sequence_number
```

We'd then need a `UNIQ [namespace, shard_id]` index. 

Lemme know your thoughts; if you'd like to keep it JSON i'm good w/ that. If you feel like a 3-col table is interesting then thats good too. I'll defer to you on the final direction 👍

Reply:
I think you are right probably it will not change and a traditional table will simplify the code so I made the changes, I have used "sequence_number" column as text because it is treated all over the lib like a  string so to avoid casting I have also defined as text. I think it wouldnt be a problem because we dont need indexing there. @harlow  please vallidate the changes in https://github.com/harlow/kinesis-consumer/pull/55/commits/ed6dcc5309b431617399a03c5f441f6462abffbf

Thank you!

Reply:
After all I have changed the column "sequence_number" to numeric since it is compatible with string when using "github.com/lib/pq"